docs/release-notes/2.4.0.md

```diff
@@ -1,3 +1,33 @@
+We are pleased to announce the release of API Gateway 2.4.0, the biggest release since the release 2.0.0 which introduced the modular architecture.
+This release introduces the new, entirely Istio-based `v2alpha1` version of the APIRule CustomResourceDefinition (CRD) and custom resource (CR).
+The new version of the APIRule CRD is a significant step towards a stable API Gateway API.
+
+## The Status of the `APIRule` CRD in Version `v1beta1`
+
+The APIRule CR in version `v1beta1` will become deprecated in 2024. While both versions are currently supported, take into consideration that you'll be required to perform some manual migration steps in the following months. If you use the `no_auth` or `jwt` access strategies, you will have to change the configuration of your APIRule CR from `v1beta1` to the respective one in version `v2alpha1`. If you use the access strategies that rely on Ory Oathkeeper or `allow`, you'll be required to migrate to the `jwt` access strategy or `noAuth`. The access strategies that are not supported in the new version are:
```

Comment:
> If you use the access strategies that rely on Ory Oathkeeper or `allow`, you'll be required to migrate to the `jwt` access strategy or `noAuth`

This sentence is a bit misleading. it would be nice to mention that this version of APIRule is not complete yet and we plan to work on extAuth handler that might be migration path to ORY Oathkeeper based oauth2 handlers. `jwt` and `allow` can be already migrated to new APIRule. 

Below list is also valid but it is misleading w/o sentence that we plan to support oauth2 handler with extAuth.